features/shipping/managing_shipments/browsing_shipments.feature

```diff
@@ -0,0 +1,29 @@
+@managing_shipments
+Feature: Browsing shipments
+    In order to manage all shipments regardlessly of orders
+    As an Administrator
+    I want to browse all shipments in the system
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "UPS" shipping method with "$10.00" fee
+        And the store has a product "Apple"
+        And the store has a product "Banana"
+        And the store allows paying with "Cash on Delivery"
+        And there is a customer "donald@duck.com" that placed an order "#00000001"
+        And the customer bought a single "Apple"
+        And the customer "Donald Duck" addressed it to "Elm street", "90802" "Duckburg" in the "United States" with identical billing address
+        And the customer chose "UPS" shipping method with "Cash on Delivery" payment
+        And this order has already been shipped
+        And there is a customer "iron@man.com" that placed an order "#00000002"
+        And the customer bought a single "Banana"
+        And the customer "Tony Stark" addressed it to "Rich street", "90802" "New York" in the "United States" with identical billing address
+        And the customer chose "UPS" shipping method with "Cash on Delivery" payment
+        And I am logged in as an administrator
```

Comment:
IMO 14 lines of background is too much for browsing 2 shipments 💃

Reply:
Agreed one more time, however as with [this case](https://github.com/Sylius/Sylius/pull/10233#discussion_r268520446) I would just open a separate PR, focusing strictly on setup orders step refactoring. It would be useful for both this and #10233 and probably other PRs as well 🖖